homeassistant/components/decora_wifi/light.py

```diff
@@ -86,6 +87,7 @@ class DecoraWifiLight(LightEntity):
     def __init__(self, switch):
         """Initialize the switch."""
         self._switch = switch
+        self._unique_id = f"{DOMAIN}_{switch}"
```

Comment:
The `switch` variable is an object in this context?

Reply:
oh I an not sure, I thougt that it would return the name but as it seems that I should pass this - "self_switch.name" ?

Reply:
You can't use a name as a UID.
See: <https://developers.home-assistant.io/docs/entity_registry_index/#unique-id-requirements>